File: src/main/java/com/iflat/wip/bean/SrOsBidding.java
package com.iflat.wip.bean;

import java.util.Date;

/**
 * Created by tyriv on 2016/9/1.
 */
public class SrOsBidding {

    private String id;
    private String pid;
    private String vendor;
    private String vendorType;
    private double amount;
    private double amountAdj;
    private String comment;
    private String creatorAcc;
    private String creatorName;
    private Date createTime;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPid() {
        return pid;
    }

    public void setPid(String pid) {
        this.pid = pid;
    }

    public String getVendor() {
        return vendor;
    }

    public void setVendor(String vendor) {
        this.vendor = vendor;
    }

    public String getVendorType() {
        return vendorType;
    }

    public void setVendorType(String vendorType) {
        this.vendorType = vendorType;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public double getAmountAdj() {
        return amountAdj;
    }

    public void setAmountAdj(double amountAdj) {
        this.amountAdj = amountAdj;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public String getCreatorAcc() {
        return creatorAcc;
    }

    public void setCreatorAcc(String creatorAcc) {
        this.creatorAcc = creatorAcc;
    }

    public String getCreatorName() {
        return creatorName;
    }

    public void setCreatorName(String creatorName) {
        this.creatorName = creatorName;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }
}

File: web/app/model/wip/SrOsBidding.js
Ext.define('iFlat.model.wip.SrOsBidding', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'srOsBidding.id', mapping: 'id', type: 'string'},
        {name: 'srOsBidding.pid', mapping: 'pid', type: 'string'},
        {name: 'srOsBidding.vendor', mapping: 'vendor', type: 'string'},
        {name: 'srOsBidding.vendorType', mapping: 'vendorType', type: 'string'},
        {name: 'srOsBidding.amount', mapping: 'amount', type: 'number'},
        {name: 'srOsBidding.amountAdj', mapping: 'amountAdj', type: 'number'},
        {name: 'srOsBidding.comment', mapping: 'comment', type: 'string'},
        {name: 'srOsBidding.creatorAcc', mapping: 'creatorAcc', type: 'string'},
        {name: 'srOsBidding.creatorName', mapping: 'creatorName', type: 'string'},
        {name: 'srOsBidding.createTime', mapping: 'createTime', type: 'string'},
    ]
});